services/app/src/helpers/getAvatarSizes.js

```diff
@@ -0,0 +1,27 @@
+export const getAvatarPxSize = (size) => {
+  if (size === 'x-small') {
+    return 26;
+  }
+  if (size === 'small') {
+    return 36;
+  }
+  if (size === 'large') {
+    return 96;
+  }
+
+  return 40;
+}
+
+export const getAvatarFontSize = (size) => {
```

Comment:
Why this returns string in pixels, while the one above returns number?

Reply:
Because antd avatar takes size prop in integer values and font size goes to style attribute, so it needs to be a string